src/reducers/workspaceInfo.ts

```diff
@@ -0,0 +1,32 @@
+import { Reducer } from 'react'
+import { SET_DOCUMENT_INFO } from '../actions'
+
+export interface WorkspaceInfo {
+    workspaceName: string | null
+    feedUri: string | null
+    readOnly: boolean | null
+}
+
+export interface SetWorkspaceInfoAction {
+    type: 'SET_DOCUMENT_INFO'
+    workspaceInfo: WorkspaceInfo
+}
+const isSetWorkspaceInfoAction = (x: any): x is SetWorkspaceInfoAction => (
+    x.type === SET_DOCUMENT_INFO
```

Comment:
Change string to 'SET_WORKSPACE_INFO'?

Reply:
This is actually to be deleted. I think it is gone in my current working branch.